manifests/init.pp

```diff
@@ -117,5 +70,4 @@
   class { '::ntp::config': } ~>
   class { '::ntp::service': } ->
   anchor { 'ntp::end': }
```

Comment:
are these anchors still required, given the existence of `contain`? (If so I would consider that a bug/need for enhancement in `contain`) [contain function doc](https://docs.puppet.com/puppet/latest/reference/lang_containment.html#the-contain-function)